src/server/verification/processor/EnrollmentProcessor.js

```diff
@@ -82,15 +82,15 @@ class EnrollmentProcessor {
 
     try {
       const isDisposing = await storage.hasTasksQueued(DISPOSE_ENROLLMENTS_TASK, {
-        subject: enrollmentIdentifier
+        subject: { enrollmentIdentifier, executeAt: 'account-removal' }
       })
 
       log.info('Got disposal state for enrollment', { enrollmentIdentifier, isDisposing })
       return isDisposing
     } catch (exception) {
       const error = exception.message
 
-      log.warn("Coundn't check disposal state for enrollment", { enrollmentIdentifier, error })
+      log.warn("Couldn't check disposal state for enrollment", { enrollmentIdentifier, error })
```

Comment:
tnx for the typos fixes

Reply:
you're welcome
check this out: 
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker